Rakefile

```diff
@@ -1,17 +1,26 @@
-require "logstash/devutils/rake"
+
+# encoding: utf-8
+require "jars/installer"
+require "fileutils"
 
 task :default do
-  system('rake -T')
+  system('rake -vT')
+end
+
+task :vendor do
+  exit(1) unless system './gradlew vendor'
 end
 
-require 'jars/installer'
-task :install_jars do
-  # If we don't have these env variables set, jar-dependencies will
-  # download the jars and place it in $PWD/lib/. We actually want them in
-  # $PWD/vendor
-  ENV['JARS_HOME'] = Dir.pwd + "/vendor/jar-dependencies/runtime-jars"
-  ENV['JARS_VENDOR'] = "false"
-  Jars::Installer.new.vendor_jars!(false)
+task :clean do
+  ["vendor/jar-dependencies", "Gemfile.lock"].each do |p|
+    FileUtils.rm_rf(p)
+  end
 end
 
-task :vendor => :install_jars
+task :test do
+  require 'rspec/core/runner'
+  require 'rspec'
+  system './gradlew clean test'
```

Comment:
suggested to do exit(1) unless system ... ensures an early exit.

Reply:
test is calling test ? ... isn't this a loop ? 

if you don't need test , you can place a dependency on the clean task

Reply:
I'm just going to remove the whole section for now - there are no Java tests in the system, so its not strictly necessary at this point